pebblo/app/pebblo-ui/src/pages/appDetailsPage.js

```diff
@@ -38,6 +46,40 @@ const DialogBody = () => {
   `;
 };
 
+const DeleteAppDialogBody = () => {
+  window.addEventListener(LOAD, function () {
+    const delete_icon = document.getElementById("delete_confirm_btn");
+    const DIALOG = document.getElementById("delete_app_dialog");
+    delete_icon?.addEventListener(CLICK, async function () {
+      const res = await DELETE_APP(
+        `${DELETE_APP_ROUTE}?app_name=${APP_DATA?.name}`
+      );
+      if (res.status === SUCCESS_CODE) {
+        window.location.href = DASHBOARD_ROUTE;
+      } else {
+        DIALOG.close();
```

Comment:
Instead of silently closing on an error, it's better to show a notification or alert that the deletion failed.

Reply:
+1

Reply:
Added toast to display both success and failure messages on deleting an application